src/index.js

```diff
@@ -61,9 +63,36 @@ export default ({
     throw new Error('Unexpected state.');
   };
 
+  const getScriptPath = (path: Object, state: Object) => {
+    const opts = state.opts;
+    const fullPath = state.file.opts.filename;
+    const basename = Path.basename(fullPath);
+    const basedir = Path.dirname(fullPath);
+    const {column, line} = path.node.loc.start;
+
+    if (opts.scriptPath) {
+      if (basename === 'index.js') {
+        return '[' + Path.basename(basedir) + '/' + basename + ':' + line + ']';
+      } else {
+        return '[' + basename + ':' + line + ']';
+      }
+    } else if (opts.fullScriptPath) {
+      const pkgDir = pkgUp.sync(basedir);
+      const relativeFilename = Path.relative(Path.dirname(pkgDir), fullPath);
+
+      return {
+        scriptColumn: t.numericLiteral(column),
```

Comment:
Whats the reason for representing this as an object, as opposed to a single string `scriptPath:line:column`?

> I know thats what I do at https://github.com/gajus/babel-plugin-log-deprecated
> though I am not necessarily a good developer to follow by example. :-(

Representing it as a single string will enable you to click–open in IDE the file at a specified line, column (e.g. https://coderwall.com/p/0kapug/iterm2-open-text-file-paths-in-sublime-text-with-jump-to-line-number)